src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java

```diff
@@ -0,0 +1,40 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+
+public class AdsVelesObject extends AdsBaseObject {
+
+  private static final String INCONTENT_VIDEO = ".video-display-wrapper";
+
+  public static final String SECOND_ARTICLE_HEADER_OASIS = "#Player_above";
+  public static final String FIRST_ARTICLE_HEADER_MOBILE = "#Header";
+
+  public static final String BIDDER_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_with_fallback_bid.*";
+  public static String DIRECT_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_with_direct.*";
+  public static String NO_OFFER_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_without_offer.*";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make NO_OFFER_PLAYER_EVENT_PATTERN a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this "public static NO_OFFER_PLAYER_EVENT_PATTERN" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1444)